### PR TITLE
Ann

### PR DESCRIPTION
**Pull request by AnnGlad**

### CODE REVIEW

File: src/main/java/com/softserve/task4/models/Human.java [hunk 1]
```diff
@@ -75,14 +86,27 @@ public void setWeight(float weight) {
         this.weight = weight;
     }
 
+    public String whatGender() {
```
Comment (AliceIgorevna) [inline src/main/java/com/softserve/task4/models/Human.java:89]: "defineGender" sound more conventionally :)

File: src/main/java/com/softserve/task4/models/Human.java [hunk 2]
```diff
@@ -75,14 +86,27 @@ public void setWeight(float weight) {
...
+        return this.getGender() ? "male" : "female";
+    }
+
+    public static String capitalizeFirstLetter(String str) {
+        return str.substring(0, 1).toUpperCase() + str.substring(1).toLowerCase();
+    }
```
Comment (AliceIgorevna) [inline src/main/java/com/softserve/task4/models/Human.java:95]: methods like this a kind of "util" ones. Maybe would be better not to place them in a Human class?

**Review by AliceIgorevna — COMMENTED** [review]
Generally approved. Can be merged once other team member gives approval

**Comment by GalynaUdaltsova** [comment]
Unnecessary branch

**Pull request closed without merging**